test/integration/sugar-mixins.spec.js

```diff
@@ -11,13 +12,11 @@ describe('Sugar mixins', () => {
     config.mixins = [{
       types: 'Global',
       extend: {
-        createAppAndLoad(appName, scriptName) {
+        createAppAndLoad(appName, loadScript) {
           return qixGlobal.createApp(appName).then(appInfo =>
```

Comment:
I'm doing that as well, but I want to keep the PRs as manageable (small and isolated) as possible. Consider it a courtesy to all of you reviewers!  :)